docs/content/manual/harvester-rancher/4678-load-balancer-different-roles-pool.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Create load balancer on different roles pools in RKE2 cluster
+---
+
+* Related issues: [#4678](https://github.com/harvester/harvester/issues/4678) [BUG] Unable to successfully create LoadBalancer service from Guest cluster post 1.2.1 upgrade
+
+
+
+## Category: 
+* Rancher 
+
+## Verification Steps
+1. Prepare three nodes Harvester and import with Rancher
+1. Provision a RKE2 guest cluster in Rancher
+1. Create the first pool, specify the `etcd` and `control plane` roles
+1. Create the second pool, specify the `work` role only
```

Comment:
@albin, do we have a test case with 2 pool created while testing guest cluster?

Reply:
After checking and helped by Albin, currently in the backend e2e test in `test_9_rancher_integration.py` 
We provide creating 3 nodes in the same roles `ALL` and in the same pool. 

Thus it seems the manual test is somehow differ with the backend e2e Rancher integration test.